ambry-clustermap/src/main/java/com.github.ambry.clustermap/Replica.java

```diff
@@ -32,6 +32,7 @@
 class Replica implements ReplicaId {
   private final Partition partition;
   private Disk disk;
+  private volatile boolean isStopped = false;
```

Comment:
there is probably no point having this here unless there is a setter method, right?

Reply:
A setter method is added for future use.